ZenPacks/zenoss/LinuxMonitor/zenpack.yaml

```diff
@@ -1894,8 +1894,30 @@ device_classes:
                                 rrdmin: 0
                                 aliases:
                                     disk_weighted_busy__pct: "10,/"
-
+                    utilizationReport:
+                        type: COMMAND
+                        usessh: true
+                        commandTemplate: "/usr/bin/env iostat -dmx -N -p 10 2"
```

Comment:
`iostat` isn't an appropriate way to collect this data. It has a number of downsides listed below in descending order of seriousness compared to parsing `/proc/diskstats` directly.

1. The values presented are only sampled from that 10 second period rather than since the last collection.
2. We're already parsing `/proc/diskstats` in the diskstats datasource.
3. It will take 10 seconds to run due to the 10/2 interval/count parameters.
4. The iostat command may not be installed.

Reply:
Hi Chet, thanks for the review. According to Anne's comment on https://jira.zenoss.com/browse/SVC-1929 I delete this PR.